tests/functional/pages/surveys/question.page.js

```diff
@@ -37,7 +37,7 @@ class QuestionPage {
   }
 
   getDisplayedName() {
-    return browser.getText('h1[class="section__title saturn"]')
+    return browser.getText('p[class="section__title neptune"]')
```

Comment:
Another styling hook—although this issue was already in the code it's a good time to sort it out.